features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/Recorder.kt

```diff
@@ -6,20 +6,17 @@
 
 package com.datadog.android.sessionreplay
 
-import androidx.annotation.MainThread
 import com.datadog.tools.annotation.NoOpImplementation
 
 @NoOpImplementation
 internal interface Recorder {
-    @MainThread
```

Comment:
why do we remove them?

Reply:
because the way I implemented them in the recorder allow them to be called on any thread. In the recorder they are being passed through an `uiThread` handler